app/collections/preview.cjsx

```diff
@@ -28,17 +28,18 @@ module.exports = React.createClass
       @setState {subjects}
 
   render: ->
-    [owner, name] = @props.collection.slug.split('/')
-
+    baseLink = ""
+    if @props.project?
+      baseLink += "projects/#{@props.project.slug}"
     <div className="collection-preview">
       <div className="collection">
         <p className="title">
-          <Link to="/collections/#{owner}/#{name}">
+          <Link to="#{baseLink}/collections/#{@props.collection.slug}">
```

Comment:
Unfortunately, this link can't be relative.

For instance on [subject pages](https://maintain-project-context-v2.pfe-preview.zooniverse.org/projects/brian-testing/ghosts/talk/subjects/3154) the preview is in the "Collections" section and builds a URL like `/projects/brian-testing/ghosts/talk/subjects/projects/brian-testing/ghosts/collections/parrish/test2`.  Wasn't this `/` prefixed a while ago?